.github/pull_request_template.md

```diff
@@ -1,6 +1,6 @@
 **Ticket:** DOC-<num>
 
-**Site:** [Staging branch](http://docs-<hotfix|feature|release>-<ver>-doc-<num>.staging.tiny.cloud/docs/tinymce/latest/)
+**Site:** [Staging branch](https://pr-<PR#>.tinymce-docs.iad.staging.tiny.cloud/docs/tinymce/8/)
```

Comment:
I believe the below should be pointing to `/latest` for this version.
```suggestion
**Site:** [Staging branch](https://pr-<PR#>.tinymce-docs.iad.staging.tiny.cloud/docs/tinymce/latest/)
```

Reply:
Due to the redirects now being built-in now pointing at 8 works fine for the preview and it will be future proof when TinyMCE 9 comes out.